### PR TITLE
Add ability to start/stop sites by name (not just ID)

### PR DESCRIPTION
**Pull request by eduwass**
This PR adds the ability to start/stop sites by using their name in the CLI. 

It'll compare the input with Local's `sites.json` file to see if there's a match by name, and return the ID if so.

Also added some basic error handling when the GraphQL Request is not successful.

### CODE REVIEW

File: src/helpers/get-site-id.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import * as fs from 'fs-extra'
+import untildify = require('untildify');
+
+export default function getSiteId(siteName: string) {
+  const sitesJsonFile = untildify('~/Library/Application Support/Local/sites.json')
```
Comment (octalmage) [inline src/helpers/get-site-id.ts:5]: Let me see if I can make this Linux friendly!

**Review by octalmage — COMMENTED** [review]
Appreciate the changes! Let me run this on my Linux box and see if I can get it working there.

**Comment by octalmage** [comment]
I realized the rest of the CLI isn't Linux or OS friendly, so I'm just going to merge this! Thanks!

**Comment by digamber89** [comment]
Hi Guys, 
Is this ability added ? Can i start a site via site-name ?

**Comment by flegfleg** [comment]
Does not work for me.

```
local-cli start-site mysite
    Error: Cannot read property 'id' of null: {"response":{"errors":[{"message":"Cannot read
    property 'id' of
    null","locations":[{"line":3,"column":9}],"path":["startSite"],"extensions":{"code":"INTERN
    AL_SERVER_ERROR"}}],"data":{"startSite":null},"status":200},"request":{"query":"\n
    mutation ($siteID: ID!) {\n        startSite(id: $siteID) {\n          id\n          name\n
              status\n        }\n      }\n    ","variables":{"siteID":"mysite"}}}```

**Pull request merged**